torchsparse/sparse_tensor.py

```diff
@@ -13,7 +14,15 @@ def __init__(self, feats, coords, stride=1):
 
     def check(self):
         if self.s not in self.coord_maps:
-            self.coord_maps[self.s] = self.C
+            if isinstance(self.s, int):
+                self.coord_maps[self.s] = self.C
+            else:
+                assert isinstance(self.s, torch.Tensor)
+                if len(self.s.shape) > 1:
+                    key = str(self.s[0].cpu().int().numpy())
+                else:
+                    key = str(self.s.cpu().int().numpy())
+                self.coord_maps[key] = self.C
```

Comment:
Similar here. We can also convert `stride` in `SparseTensor` to a tuple of integers to avoid these checks.

Reply:
Good idea. I've implemented that.